app/uk/gov/hmrc/soletraderidentificationfrontend/services/RegistrationOrchestrationService.scala

```diff
@@ -30,12 +30,12 @@ class RegistrationOrchestrationService @Inject()(soleTraderIdentificationService
                                                  auditService: AuditService
                                                 )(implicit ec: ExecutionContext) {
 
-  def register(journeyId: String)(implicit hc: HeaderCarrier): Future[RegistrationStatus] = for {
+  def register(journeyId: String, journeyConfig: JourneyConfig)(implicit hc: HeaderCarrier): Future[RegistrationStatus] = for {
```

Comment:
I'd be tempted to keep the two public method signatures consistent and use register: String here rather than journeyConfig: JourneyConfig.

Reply:
done